src/App.tsx

```diff
@@ -1,8 +1,15 @@
-import './App.css';
+import React from 'react';
 
 function App() {
   return (
-    <div className="App">
+    <div style={{ textAlign: 'center' }}>
```

Comment:
Could you create a separate css style and move inline style to there? It is a bad practice to write inline styles and declines readability